aw_server/rest.py

```diff
@@ -152,7 +152,16 @@ class EventsResource(Resource):
     @copy_doc(ServerAPI.get_events)
     def get(self, bucket_id):
         args = request.args
-        limit = int(args["limit"]) if "limit" in args else 100
+        
+        if "limit" in args:
+            limit = int(args["limit"])
+        elif "start" in args:
```

Comment:
I don't think this is makes much sense from an API standpoint, a default should only ever have a single value no matter what other parameters you have specified.

I would prefer if the default limit was -1 instead actually, there are very few who actually use this API so I think it should be fine to break backwards compatibility.

Reply:
I think I understand what you're saying. 
I will go back and modify it so when the "limit" is not assigned a value, it will default to '-1'.